packages/frontend/app/(dashboard)/course/[cid]/insights/components/InsightCard.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+type InsightCardProps = {
+  children: React.ReactNode
+  title: string
+  description?: string
+}
+
+const InsightCard: React.FC<InsightCardProps> = ({
+  children,
+  title,
+  description,
+}) => {
+  return (
+    <div className="border-b-helpmeblue-light hover:border-b-helpmeblue flex-[1_1_auto] rounded-lg border-b-2 bg-gray-50 p-8 drop-shadow-md transition-all">
+      <h1>{title}</h1>
```

Comment:
h1 might be too high of a heading? I believe we're only supposed to have one h1 per page for page semantics, so h2 might be a better pick

Reply:
addressed by making it a <b className={'text-lg'}> tag, rather than a heading